pom.xml

```diff
@@ -82,7 +82,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-job</artifactId>
-            <scope>test</scope>
+            <version>2.25</version>
```

Comment:
Is there a good reason why the scope of this has changed from `test`? Same with the other dependencies. They should be `test`.